sawyer_gazebo/launch/sawyer_world.launch

```diff
@@ -9,8 +9,24 @@
   <arg name="debug" default="false"/>
   <arg name="head_display_img" default="$(find sawyer_gazebo)/share/images/sawyer_sdk_research.png"/>
 
-  <!-- These arguments load the electric grippers, for example electric_gripper:=true -->
+  <!-- This argument loads the electric gripper, for example electric_gripper:=true -->
   <arg name="electric_gripper" default="false"/>
+  <!-- This argument loads sawyer's pedestal URDF -->
+  <arg name="pedestal" default="true"/>
+  <!-- This argument fixes the robot statically to the world -->
+  <arg name="static" default="true"/>
+  <!-- This argument fixes the robot statically to the world -->
+  <arg name="load_gazebo" default="true"/>
+  <!-- This argument sets the initial joint states -->
+  <arg name="initial_joint_states"
```

Comment:
Where did these values come from and can they at least be rounded to one or two decimals?